Submitting updated 1_ssh.md -- updated based on comments

clstmbrly opened the pull request:
@lachellel This is the updated 1_ssh.md (Userconfig collection).  Changes have been made based on your comments.  (Note that the older ssh.md version has been deleted from the Devconfig collection.)  I was surprised to see the long list of past commits come up with this P.R., so if there is a problem, please let me know.  

There is one rendering issue with the very last code block.  Couldn't find a way to create a mandatory line break within the block. 

If you have additional comments or changes, please let us know.  Thanks very much and have a great Labor Day weekend:-).

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

lachellel committed:
Merge pull request #122 from clstmbrly/ssh

Enable PIV/CAC for SSH Procedures (Windows)

godadada committed:
Create ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

godadada committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

clstmbrly committed:
Update ssh-linux.md

lachellel committed:
Merge pull request #125 from clstmbrly/ssh-linux.md

2 changes to command line instructions.

clstmbrly committed:
Add current ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Update ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Update ssh-all-2.md

Web-rendered version using Jekyll/Atom and Kramdown.

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Update ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Delete ssh-all.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Update ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Create ssh-all-2.md

clstmbrly committed:
Merge pull request #138 from clstmbrly/ssh-all

File:  Ssh-all-2.md @lachellel I mistakenly thought you had denied the previous ssh-all-2.md P.R.  After I created a new P.R. (6/13), I realized that the old one was still open.  I couldn't find a way to close the new P.R., so I selected the "Merge" option.  I hope this worked okay on your side.  Please let me know if any corrective action is needed.  Your requested changes have been added, as well as some additional editorial changes for readability, etc.  Please let me know if you have additional changes.  Thank you.

clstmbrly committed:
Delete ssh.md

clstmbrly committed:
Delete ssh-linux.md

clstmbrly committed:
Create ssh.md

clstmbrly committed:
Delete ssh-all-2.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update index.md

clstmbrly committed:
Create 0_Index.md

clstmbrly committed:
Update 0_Index.md

clstmbrly committed:
Delete index.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

clstmbrly committed:
Update ssh.md

lachellel committed:
Merge pull request #150 from clstmbrly/ssh-all

Updated ssh.md and new 0_Index.md

godadada committed:
Create 1_ssh.md

godadada committed:
Update 1_ssh.md

godadada committed:
Update 1_ssh.md

godadada committed:
Update 1_ssh.md

clstmbrly committed:
Update 1_ssh.md

clstmbrly updated 1_ssh.md - 8/30/2017

clstmbrly committed:
Update 1_ssh.md

Protiviti-JSargent committed:
troubleshooting permalink for live-preview

Protiviti-JSargent committed:
troubleshooting permalink for live-preview

clstmbrly committed:
Update 1_ssh.md

clstmbrly committed:
Update 1_ssh.md

clstmbrly committed:
Update 1_ssh.md

clstmbrly committed:
Update 1_ssh.md

Attempt to create mandatory break in code block

clstmbrly committed:
Delete ssh.md

Deleting older version of ssh.md.  This file was moved to GSA/PIV-Guides/ssh/userconfig/1_ssh.md and updated.

clstmbrly committed:
Update 1_ssh.md

clstmbrly committed:
Update 1_ssh.md

lachellel committed:
updated intro

lachelle rewrite
active voice

lachellel committed:
Merge branch 'staging' into ssh

lachellel committed:
rearranging ssh and checking steps for latest version - updates required

lachellel commented:
we walked through the steps and the latest versions are very different.  new versions pushed...we still have some changes...

clstmbrly committed:
Update 02_ssh.md

Minor editorial  changes.

clstmbrly committed:
Update 02_ssh.md

Minor editorial changes and <TODO> follow-up questions re: macOS X Sierra and Pageant.

clstmbrly committed:
Update 02_ssh.md

Fixed PuTTY version and added <TODO> comments.

clstmbrly committed:
Update 02_ssh.md

Follow-up comments to questions from 10/5 meeting

clstmbrly committed:
Update 02_ssh.md

clstmbrly committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Add files via upload

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Add files via upload

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

lachellel committed:
small updates

clstmbrly committed:
Update 02_ssh.md

clstmbrly committed:
Update 02_ssh.md

Editorial changes with questions.

clstmbrly committed:
Update 02_ssh.md

Editorial changes

clstmbrly committed:
Update 02_ssh.md

Editorial changes

clstmbrly committed:
Update 02_ssh.md

Editorial changes and questions

indrajit-gsa committed:
Update 02_ssh.md

Modifying the write-up to include linux desktop. Cleaning up comments from write-up.

indrajit-gsa committed:
Update 02_ssh.md

Changed *nix to Linux. Added new instructions.

indrajit-gsa committed:
Update 02_ssh.md

Separated out the linux desktop and updated the OpenSC commands.

clstmbrly committed:
Update 02_ssh.md

Editorial changes

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

Updated instructions for macOS for SSH and removed login from Linux desktops for this iteration of this playbook. Changed the reference of *nix to Linux. The Unix instructions are similar but different than the Linux version.

indrajit-gsa committed:
Update 02_ssh.md

Added/responded comments during editorial review.

indrajit-gsa committed:
Update 02_ssh.md

Minor edits.

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

indrajit-gsa committed:
Update 02_ssh.md

clstmbrly committed:
Update 02_ssh.md

Editorial and content changes

clstmbrly committed:
Update 02_ssh.md

clstmbrly committed:
Update 02_ssh.md

Editorial and content changes

clstmbrly committed:
Update 02_ssh.md

Editorial and formatting changes

clstmbrly committed:
Update 02_ssh.md

Rendering fixes

clstmbrly committed:
Update 02_ssh.md

Rendering

clstmbrly committed:
Update 02_ssh.md

Rendering

clstmbrly committed:
Update 02_ssh.md

Rendering

clstmbrly committed:
Update 02_ssh.md

Rendering

clstmbrly committed:
Update 02_ssh.md

lachellel committed:
updating index

lachellel committed:
delete index in dev

delete index in dev

lachellel committed:
making alerts content 

updating alerts to use content over heading

lachellel committed:
clean title

remove piv/cac
